.eslintrc.json

```diff
@@ -12,7 +12,8 @@
     "@typescript-eslint/no-unused-vars": [
       "error",
       {
-        "ignoreRestSiblings": true
+        "ignoreRestSiblings": true,
+        "destructuredArrayIgnorePattern": "^_"
```

Comment:
Added this because `validSearchParamsObject `uses `([_, value]) => value !== undefined,` which for some reason isn't covered by `ignoreRestSiblings`